libs/datasets/combined_datasets.py

```diff
@@ -0,0 +1,173 @@
+from typing import Dict, Type, List, NewType
+import functools
+import pandas as pd
+from libs.datasets import dataset_utils
+from libs.datasets import dataset_base
+from libs.datasets import data_source
+from libs.datasets.timeseries import TimeseriesDataset
+from libs.datasets.latest_values_dataset import LatestValuesDataset
+from libs.datasets.sources.jhu_dataset import JHUDataset
+from libs.datasets.sources.nha_hospitalization import NevadaHospitalAssociationData
+from libs.datasets.sources.cds_dataset import CDSDataset
+from libs.datasets.sources.covid_tracking_source import CovidTrackingDataSource
+from libs.datasets.sources.covid_care_map import CovidCareMapBeds
+from libs.datasets.sources.fips_population import FIPSPopulation
+from libs.datasets import CommonFields
+from libs.datasets import dataset_filter
+from libs import us_state_abbrev
+
+FeatureDataSourceMap = NewType(
```

Comment:
As a whole, this is a big step in the right direction.  As you've clearly outlined here, with this data structure, we can see the original sources for a given field.

For the purposes of visibility, it's probably best to note that we've discussed that one way to improve this would be to move from showcasing datasets dependencies to showing casing a dependency graph of transformations.

Reply:
Yeah that's a great idea -  I'll add a comment around that